leadbutt.py

```diff
@@ -85,6 +87,170 @@ def get_options(config_options, local_options, cli_options):
+    list_map = {
+        "network": "interface",
+        "diskIO": "device",
+        "fileSys": "name",
+        "processList": "name"
+    }
+    # return list_map.get(statistic_dict[category], index)
+    return statistic_dict[list_map.get(category)]
+
+
+def unit_type_map(category, statistic):
+    unit_map = {
+        "cpuUtilization": {
+            "guest": "Percent",
+            "irq": "Percent",
+            "system": "Percent",
+            "wait": "Percent",
+            "idle": "Percent",
+            "user": "Percent",
+            "total": "Percent",
+            "steal": "Percent",
+            "nice": "Percent",
+        },
+        "loadAverageMinute": {
+            "fifteen": "Count",
+            "five": "Count",
+            "one": "Count",
+        },
+        "memory": {
+            "writeback": "Kilobytes",
+            "hugePagesFree": "Count",
+            "hugePagesRsvd": "Count",
+            "hugePagesSurp": "Count",
+            "hugePagesTotal": "Count",
+            "cached": "Kilobytes",
+            "hugePagesSize": "Kilobytes",
+            "pageTables": "Kilobytes",
+            "dirty": "Kilobytes",
+            "mapped": "Kilobytes",
+            "active": "Kilobytes",
+            "total": "Kilobytes",
+            "slab": "Kilobytes",
+            "buffers": "Kilobytes",
+        },
+        "tasks": {
+            "sleeping": "Count",
+            "zombie": "Count",
+            "running": "Count",
+            "stopped": "Count",
+            "total": "Count",
+            "blocked": "Count"
+        },
+        "swap": {
+            "cached": "Kilobytes",
+            "total": "Kilobytes",
+            "free": "Kilobytes"
+        },
+        "network":
+        {
+            "rx": "Bytes/Second",
+            "tx": "Bytes/Second"
+        },
+        "diskIO":
+        {
+            "writeKbPS": "Kilobytes/Second",
+            "readIOsPS": "IO/Second",
+            "await": "Milliseconds",
+            "readKbPS": "Kilobytes/Second",
+            "rrqmPS": "IO/Second",
+            "util": "Percent",
+            "avgQueueLen": "Milliseconds",
+            "tps": "Count",
+            "readKb": "Kilobytes",
+            "writeKb": "Kilobytes",
+            "avgReqSz": "Kilobytes",
+            "wrqmPS": "IO/Second",
+            "writeIOsPS": "IO/Second"
+        },
+        "fileSys":
+        {
+            "used": "Kilobytes",
+            "usedFiles": "Count",
+            "usedFilePercent": "Percent",
+            "maxFiles": "Count",
+            "total": "Kilobytes",
+            "usedPercent": "Percent"
+        },
+        "processList":
+        {
+            "vss": "Kilobytes",
+            "tgid": "Count",
+            "parentID": "Count",
+            "memoryUsedPc": "Percent",
+            "cpuUsedPc": "Percent",
+            "id": "Count",
+            "rss": "Kilobytes"
+        }
+    }
+    return unit_map[category].get(statistic, 'Count')
```

Comment:
The way the code is written, you are re-creating the same hard-coded multi-level dictionary every time you call this function. Instead of creating a function, create this dictionary as a constant and get the data.

```python
UNIT_MAP = {
...
}

...
...

foo = UNIT_MAP['bar']  # rather than unit_type_map('bar', 'baz')
```